design/index.html

```diff
@@ -0,0 +1,112 @@
+---
+layout: main
+title: Design
+---
+
+{% capture title %}
+Learn design best practices
+{% endcapture %}
+
+{% capture text %}
+Use these resources to address user needs by focusing on user centred design, research and techniques.
+{% endcapture %}
+
+{% include digital_element_header.html title=title text=text %}
+
+{% capture content %}
+
+<div class="container col-md-10 text-center">
+  <h2>Develop your knowledge</h2>
+  <p class="lead">
+    Resources for improving your research and design skills, lovingly crafted by the team at Made Tech.
+  </p>
+</div>
+
+<div class="row">
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">User research</h3>
+      <p class="card-text">
+        Learn various techniques on how to conduct iterative user research effectively.
+      </p>
+      <a href="/" class="btn btn-success">Learn more</a>
+    </div>
+  </div>
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">User stories</h3>
+      <p class="card-text">
+        Understand how to frame problems to work towards implementing features.
+      </p>
+      <a href="/" class="btn btn-success">Learn more</a>
+    </div>
+  </div>
+</div>
+<div class="row">
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">Personas</h3>
+      <p class="card-text">
+        Create fictional characters to identify what your users needs so your designs are user centred.
+      </p>
+      <a href="/" class="btn btn-success">Learn more</a>
+    </div>
+  </div>
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">Prototyping</h3>
+      <p class="card-text">
+        Develop design thinking skills by regularly testing ideas quickly ensuring constant improvement.
+      </p>
+      <a href="/" class="btn btn-success">Learn more</a>
+    </div>
+  </div>
+</div>
+
+<div class="container col-md-10 text-center">
+  <h2>Fundamental design skills</h2>
+  <p class="lead">
+    Coming soon!
+  </p>
+</div>
+
+<div class="row">
+  <div class="col-sm">
+    <div class="card-body card-border">
+      <h3 class="card-title mt-0">Placeholder 1</h3>
+      <p class="card-text">
+        <i>Coming soon!</i>
```

Comment:
You should only be using `<i>` for things like technical terms or specialist language, `<em>` should instead be used for emphasis.

Reply:
Resolved in https://github.com/madetech/learn/pull/160/commits/acdcca615cc64f35b2fbf44c9bf55930ba8e38c7.